app/src/main/resources/application-prod.properties

```diff
@@ -1,47 +1,26 @@
 # Readiness and Liveness
-%prod.registry.metrics.PersistenceExceptionLivenessCheck.errorThreshold=${LIVENESS_ERROR_THRESHOLD:5}
-%prod.registry.metrics.PersistenceExceptionLivenessCheck.counterResetWindowDurationSec=${LIVENESS_COUNTER_RESET:30}
-%prod.registry.metrics.PersistenceExceptionLivenessCheck.statusResetWindowDurationSec=${LIVENESS_STATUS_RESET:60}
-%prod.registry.metrics.ResponseErrorLivenessCheck.errorThreshold=${LIVENESS_ERROR_THRESHOLD:5}
-%prod.registry.metrics.ResponseErrorLivenessCheck.counterResetWindowDurationSec=${LIVENESS_COUNTER_RESET:30}
-%prod.registry.metrics.ResponseErrorLivenessCheck.statusResetWindowDurationSec=${LIVENESS_STATUS_RESET:60}
+apicurio.metrics.PersistenceExceptionLivenessCheck.errorThreshold=5
```

Comment:
Default value of this property in code is `1`, and we're overriding in **prod** with `5` and in **test** with `10000`. This is nitpicky, but is the general rule that the defaults in code are intended for **dev**, and should be overridden where needed for **prod** and **test**? Intuitively, at least for me, the defaul value should be for **prod**.

Reply:
Default values are used for all profiles unless they are overriden. I find it much simpler than having everything defined three times (note that there is no longer an `application-dev.properties`). I'm not against the other way around, but the reality is that we're overriding just a handful of properties for the prod and test profiles that can be even done in some other (in my view) less confusing ways (like test profiles).

Reply:
No preference here.  :)

Reply:
No preference here.  :)